src/application/Main.java

```diff
@@ -1,11 +1,12 @@
 package application;
 
 import application.engine.Machine;
+import application.engine.Menu;
 
 public class Main {
     public static void main(String[] args) {
-        Machine machine = new Machine();
-        machine.run();
+        Machine machine = new Machine(400, 540, 120, 9, 550);
+        Menu menu = new Menu(machine);
```

Comment:
Create Scanner in main (not in Menu) 
Pay attention that `scanner.close()` should be called in `finally` block or use try with resources
```suggestion
        try (Scanner scanner = new Scanner(System.in)) {
        ....
        }
```